tests/integrationv2/providers.py

```diff
@@ -171,13 +171,13 @@ def supports_certificate(cls, cert: Cert):
 
     @classmethod
     def supports_protocol(cls, protocol):
-        if not cls._pss_supported() and protocol == Protocols.TLS13:
+        if not cls._pss_supported() and protocol and protocol == Protocols.TLS13:
```

Comment:
Just curious why this is needed. Isn't it valid to say `None == Protocols.TLS13`?

Reply:
We define equals as the values being equal, and "None" doesn't have a value: https://github.com/aws/s2n-tls/blob/main/tests/integrationv2/common.py#L206-L207

I guess I could change that definition instead?